.github/messages.yaml

```diff
@@ -0,0 +1,17 @@
+lock-issue: |
+  Hi there, thank you for your contribution!
+
+  This issue has been **automatically locked** because it has not had recent activity after being closed.
+
+  Please open a new issue if needed.
+
+  Thanks!
+
+lock-pr: |
+  Hi there, thank you for your contribution!
+
+  This pull request has been **automatically locked** because it has not had recent activity after being closed.
+
+  Please open a new issue or pull request if needed.
+
+  Thanks!
```

Comment:
@jezdez followup (2/2) from https://github.com/conda/conda/pull/11249, here's the universal lock blurbs, let me know what you think!